src/LiveComponent/assets/dist/live_controller.js

```diff
@@ -1,3240 +1,2505 @@
-import { Controller } from '@hotwired/stimulus';
-
-class BackendRequest {
-    constructor(promise, actions, updateModels) {
-        this.isResolved = false;
-        this.promise = promise;
-        this.promise.then((response) => {
-            this.isResolved = true;
-            return response;
-        });
-        this.actions = actions;
-        this.updatedModels = updateModels;
-    }
-    containsOneOfActions(targetedActions) {
-        return this.actions.filter((action) => targetedActions.includes(action)).length > 0;
-    }
-    areAnyModelsUpdated(targetedModels) {
-        return this.updatedModels.filter((model) => targetedModels.includes(model)).length > 0;
-    }
+import { Controller } from "@hotwired/stimulus";
+import { Idiomorph } from "idiomorph/dist/idiomorph.esm.js";
```

Comment:
Should be inlined

Reply:
Fixed by moving `idiomorph` from `dependencies` to `devDependencies`, and it makes 100% sense (since this dependency was inlined, it's useless to download it when downloading this package)

Note: it was no present in `importmap` either, that's fine.